### DISCUSSION TITLE
no cluster api resources metric saw from grafana dashboard

### DISCUSSION THREAD

**Discussion by JiangHuiSSS** [kubernetes-sigs/cluster-api#10325] [General]
Hi team, I'm trying to use tilt to install all these cluster api observability tools listed in https://github.com/kubernetes-sigs/cluster-api/tree/main/hack/observability, when all tools are installed on the tkgm cluster, the kube-state-metric pod logs shows:
```
E0325 06:00:09.310828       1 registry_factory.go:682] "capi_clusterclass_status_condition" err="[status,conditions]: expected value for path to be string, got <nil>"
E0325 06:00:09.310872       1 registry_factory.go:682] "capi_clusterclass_status_condition_last_transition_time" err="[status,conditions]: got nil while resolving path"
```
and there is no cluster api resources metric saw from grafana dashboard, anyone knows why? Any help would be appreciated!:blush:
<img width="1502" alt="Screenshot 2024-03-27 at 10 51 37" src="https://github.com/kubernetes-sigs/cluster-api/assets/26076848/c1db1036-f39f-43ff-a4f1-94e3f130c1dc">

**Comment by fabriziopandini** [comment]
Deleting this thread, the same question has been asked on many channels
